packages/transformers/html/src/HTMLTransformer.js

```diff
@@ -26,6 +26,7 @@ export default (new Transformer({
   async transform({asset, options}) {
     // Handle .htm
     asset.type = 'html';
+    asset.isIsolated = true;
```

Comment:
This is now necessary to prevent asset reachability checks for linked HTML files (e.g. `<a href="other.html">`) from returning true. The bundle graph was updated to consider this field when determining where to connect the bundle group. Previously, `isEntry` was set on those dependencies, but now `needsStableName` doesn't also mean it's an entry.